src/aarch64.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+This is a part of Loops project.
+Distributed under Apache 2 license.
+See https://github.com/vpisarev/loops/LICENSE
+*/
+
+#ifndef __LOOPS_AARCH64_HPP__
+#define __LOOPS_AARCH64_HPP__
+
+#include "composer.hpp"
+#include "backend.hpp"
+#include "common.hpp"
+#include <vector>
+#include <map>
+
+namespace loops
+{
+enum {
+    A64_LDRSW = 0,
+    A64_STR32_I,
+    A64_MOV,
+    A64_ADD_R,
+    A64_MUL,
+    A64_SDIV,
+    A64_CMP_R,
+    A64_B_LT,
+    A64_B_LE,
+    A64_B_GT,
+    A64_B_GE,
+    A64_RET
+};
+
+class aarch64_backend : public backend_impl
```

Comment:
Let's agree on the naming conventions and stick to them. We either use MixedCaseNotation where class names start with capital letter and method names start with lowercase letter or we use lower_case_ids with underscores in the middle. Let's not mix two styles.